sanic_json_logging/__init__.py

```diff
@@ -27,13 +27,14 @@ def setup_json_logging(
     configure_task_local_storage: bool = True,
     context_var: str = "sanicjsonlogging",
     disable_json_access_log: bool = False,
+    config: dict = LOGGING_CONFIG_DEFAULTS
 ) -> None:
     """
     Sets up request logging
     """
     # Set up logging
-    LOGGING_CONFIG_DEFAULTS["formatters"]["generic"]["context"] = context_var
-    logging.config.dictConfig(LOGGING_CONFIG_DEFAULTS)
+    config["formatters"]["generic"]["context"] = context_var
```

Comment:
If a user can override the value of `config`, they could omit generic and context dict's, can you add some error checking to add those in if they don't exist.